commons/helpers/pods_helper.py

```diff
@@ -492,3 +492,16 @@ def get_deployment_name(self, num_nodes):
             resp = resp_node[i + const.NODE_INDEX].split(' ')
             deploy_list.append(resp[0])
         return deploy_list
+
+    def kill_process_in_container(self,pod_name,container_name,process_name):
```

Comment:
Spaces after ',', as we need to take care of pylint, etc warnings.

Reply:
Done.